.lgtm.yml

```diff
@@ -0,0 +1,4 @@
+path_classifiers:
+  thirdparty:
+    - /tools/amalgamate
+    - /tools/cpplint
```

Comment:
I'm not 100% sure that absolute paths are valid. It is used in one example like so: `exclude: /` (exclude everything)

We could customize path classification completely, but I don't know if anyone cares to keep this up-to-date. This just hides the `cpplint.py` alert.

The other issue was not – as I suspected – because of `single_include/` but because of the addition of the 3.10.5 header as part of the ABI test. This was fixed by simply renaming the header guard to match the file name (`json_v3_10_5.hpp`).